### PR TITLE
Additional pages display once user logged in

### PR DESCRIPTION
**Pull request by Baonganbui123**

### CODE REVIEW

File: src/main/java/page_object/BasePage.java [hunk 1]
```diff
@@ -59,9 +64,21 @@ public void clickBookTicketTab() {
...
+    public void clickMyTicketTab() {
+        getMyTicketTab().click();
+    }
+
     public Boolean isLogoutTabDisplayed() {
         return ElementHelpers.isElementDisplayed(DriverHelpers.getDriver().findElement(btnLogoutTab));
     }
+
+    public Boolean isChangePasswordTabDisplayed() {
+        return ElementHelpers.isElementDisplayed(DriverHelpers.getDriver().findElement(btnChangePasswordTab));
```
Comment (tanmle) [inline src/main/java/page_object/BasePage.java:76]: why dont you `private getPasswordTab` then reuse getPasswordTab in this method?

File: src/main/java/page_object/BasePage.java [hunk 2]
```diff
@@ -59,9 +64,21 @@ public void clickBookTicketTab() {
...
+    }
+
+    public Boolean isMyTicketTabDisplayed() {
+        return ElementHelpers.isElementDisplayed(DriverHelpers.getDriver().findElement(btnMyTicketTab));
```
Comment (tanmle) [inline src/main/java/page_object/BasePage.java:80]: same here

**Review by tanmle — CHANGES_REQUESTED** [review]

File: src/main/java/page_object/BasePage.java (hunk 1 above)
Comment (Baonganbui123) [reply src/main/java/page_object/BasePage.java]: I fixed it, please check again

File: src/main/java/page_object/BasePage.java (hunk 2 above)
Comment (Baonganbui123) [reply src/main/java/page_object/BasePage.java]: I fixed it, please check again

File: src/main/java/page_object/ChangePasswordPage.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package page_object;
+
+import helpers.DriverHelpers;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class ChangePasswordPage {
+
+    //Locators
+    private final By lblHeader = By.cssSelector("div#content > h1[align='center']");
```
Comment (MinhQuan119) [inline src/main/java/page_object/ChangePasswordPage.java:10]: Do all pages use the same cssSelector for lblHeader? Should we consider to put it in BasePage?

**Review by tanmle — APPROVED** [review]

File: src/main/java/page_object/ChangePasswordPage.java (hunk 3 above)
Comment (MinhQuan119) [reply src/main/java/page_object/ChangePasswordPage.java:10]: If yes, please take a look for its element and method too

File: src/test/java/tests/LoginTests.java [hunk 4]
```diff
@@ -55,4 +61,34 @@ public void tc005_UserEnterValidUsernameExceptPassword() {
         LogHelpers.printInfo("Verify that error when enters wrong 'Password' several times displayed");
         Assert.assertEquals(loginPage.getErrorPasswordMsg(), errorWrongPasswordSeveralTimes, "Error password message doesn't display correctly");
     }
+
+    @Test(description = "Additional pages display once user logged in")
+    public void tc006_AdditionalPagesDisplayWhenLogin(){
+        LogHelpers.printInfo("Click on 'Login' tab");
+        loginPage.clickLoginTab();
```
Comment (MinhQuan119) [inline src/test/java/tests/LoginTests.java:68]: Did it on BeforeMethod precondition

File: src/main/java/page_object/ChangePasswordPage.java (hunk 3 above)
Comment (Baonganbui123) [reply src/main/java/page_object/ChangePasswordPage.java]: I changed it, please check again

File: src/test/java/tests/LoginTests.java (hunk 4 above)
Comment (MinhQuan119) [reply src/test/java/tests/LoginTests.java:68]: Please remove it, it's duplicate

Comment (Baonganbui123) [reply src/test/java/tests/LoginTests.java]: Done

**Review by MinhQuan119 — APPROVED** [review]

**Pull request merged**